docs/decisions/0014-single-taxonomy-view-api.rst

```diff
@@ -0,0 +1,156 @@
+13. Single taxonomy view API
+=====================================
+
+
+Context
+--------
+
+This view returns tags of a closed taxonomy (for MVP has not been implemented yet
+for open taxonomies). It is necessary to make a decision about what structure the tags are going 
+to have, how the pagination is going to work and how will the search for tags be implemented.
+It was taken into account that a taxonomy can have a large number of tags and are mostly represented as trees.
```

Comment:
Please add the use cases of this API:
* "As a taxonomy administrator, I want to see all the tags available for use with a closed taxonomy, so that I can see the taxonomy's structure in the management interface."
* As a course author, when I am editing the tags of a component, I want to see all the tags available from a particular taxonomy that I can use. (**maybe**? I think we'll need to add this use case, as we discussed.)

Non-use case:
* As a content author, when searching/filtering a course/library, I want to see which tags are applied to the content and use them to refine my search. - this is _excluded_ from this API's use case because this is automatically handled by elasticsearch/opensearch.